go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/spf13/cobra v1.6.1
 	github.com/spf13/viper v1.14.0
 	github.com/stretchr/testify v1.8.1
-	github.com/tendermint/tendermint v0.34.23
+	github.com/tendermint/tendermint v0.35.0
```

Comment:
Again, @dependabot! Come on, man; this is not serious! This way of secretly bumping tendermint... You're trying to sabotage our codebase or what!?

Reply:
I think tendermint needs to retract this version